### DISCUSSION TITLE
Ho to pair a Shimano STEPS ebike?

### DISCUSSION THREAD

**Discussion by KhaliFC** [maca88/E-Bike-Edge-MultiField#8] [Q&A] [answered]
Hi, I have a Canyon Neuron:On bicycle, its a shimano STEPS EP8 ebike.

I have installed the app, configured it as a data field in a data screen in my emtb profile. 
I have removed my ebike from sensors.

I put the data screen in my garmin 530, I turn on the bike, nothing ever happens. Am I supposed to do something else? This is what I understood from the instructions.

Thank you!

**Comment by maca88** [comment, marked as answer]
Hi, based on the specification of the motor, Shimano for some reason seems to be using a private ANT communication (https://www.shimano-steps.com/e-bikes/europe/en/product-information/mtb/ep8), which isn't available to the public. This data field support only e-bikes that are using [ANT+ Light Electric Vehicle](https://www.thisisant.com/developer/ant-plus/device-profiles/#tab_527) profile for the communication.
You can try to use `Ebike Field` app by Jan Capek: https://apps.garmin.com/en-US/apps/1532f0d9-fd19-4b63-b038-435d8fd670a4#0
maybe he was able to obtain the Shimano private ANT specification and implement it in the data field.

**Reply by KhaliFC** [reply]
Hi, thank you! I will try.
